### PR TITLE
fix: invoice changes

### PR DESCRIPTION
**Pull request by Hardeep8587**
## Change Description

Environment file changes.
Invoice changes - debit credit

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: (Check only applicable)

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Hardeep8587** [comment]
@kunalfleethawks please review and merge the environment file

File: src/app/pages/accounts/chart-accounts/chart-of-accounts/chart-of-accounts.component.ts [hunk 1]
```diff
@@ -16,27 +16,28 @@ export class ChartOfAccountsComponent implements OnInit {
...
-//     amount: 200,
-//     currency: 'CAD',
-//     trxRunTotal: 0,
-//     desc: 'test desc'
-//   };
-//   this.accountService.putData(`chartAc/trx/${this.carrierID}/${this.receivedActID}`, data).subscribe((res) => {
-//   });
-// }
+trxFn() {
+  let internalID = 'ACT35';
```
Comment (kunalfleethawks) [inline src/app/pages/accounts/chart-accounts/chart-of-accounts/chart-of-accounts.component.ts:27]: remove this .

File: src/app/pages/accounts/invoices/add-invoice/add-invoice.component.ts [hunk 2]
```diff
@@ -138,27 +143,49 @@ export class AddInvoiceComponent implements OnInit {
...
+          amount: result.Items[0].HST,
+        },
+        {
+          name: 'PST',
+          amount: result.Items[0].PST,
+        },
+      ];
+    }
+      else {
+        console.log('else part of invoice');
```
Comment (kunalfleethawks) [inline src/app/pages/accounts/invoices/add-invoice/add-invoice.component.ts:168]: remove this.

File: src/app/pages/accounts/invoices/add-invoice/add-invoice.component.ts [hunk 3]
```diff
@@ -185,10 +212,26 @@ export class AddInvoiceComponent implements OnInit {
   }
 
   addInvoice() {
-    console.log('input', this.invoiceData);
+    // this.invoiceData.transactionLog[];
```
Comment (kunalfleethawks) [inline src/app/pages/accounts/invoices/add-invoice/add-invoice.component.ts:215]: remove this.

File: src/environments/environment.dev.ts [hunk 4]
```diff
@@ -9,9 +9,8 @@ export const environment = {
   isAccountsEnabled: true,
   BaseUrl: 'https://fleetservice.ap-south-1.fleethawks.com/api/v1/',
   AssetURL: 'https://fh-cloud-service-uploads.s3.us-east-2.amazonaws.com',
-  AccountServiceUrl: 'https://accounts.ap-south-1.fleethawks.com/',
-  SafetyServiceUrl : 'https://safety.ap-south-1.fleethawks.com/api/v1/',
-
+  AccountServiceUrl: ' https://accounts.ap-south-1.fleethawks.com/api/v1/',
+  SafetyUrl: 'https://safety.ap-south-1.fleethawks.com/api/v1/',
```
Comment (kunalfleethawks) [inline src/environments/environment.dev.ts:13]: Change this to SafetyServiceUrl

**Review by kunalfleethawks — APPROVED** [review]

**Pull request merged**